src/braket/experimental/autoqasm/README.md

```diff
@@ -136,10 +142,100 @@ Please tag your question with "Amazon Braket" and mention AutoQASM in the questi
 
 ## Tests
 
-To run only AutoQASM tests (and skip the rest of the unit tests), run:
+To run only AutoQASM tests (and skip the rest of the Amazon Braket SDK unit tests), run:
 ```
 tox -e unit-tests -- test/unit_test/braket/experimental/autoqasm
 ```
 
 Note that you may first need to run `pip install -e .[test]`. More information on running tests
 can be found in the [top-level README](../../../../README.md).
+
+## Frequently asked questions
+
+###  1. Will AutoQASM build a library of quantum algorithms or quantum applications?
+
+No, we are focused on AutoQASM as an interface for low-level expression of
+quantum programs: circuits, gates and pulses. Higher-level algorithm
+libraries could be implemented using AutoQASM and benefit from modular
+AutoQASM functionality such as subroutines.
+
+### 2. What is the relationship between AutoQASM and OpenQASM?
+
+AutoQASM can be seen as implementing a builder pattern for OpenQASM. It
+allows you serialize your program to OpenQASM with `Program.to_ir()`. The
+interface is not strongly tied to OpenQASM so we could serialize to other
+formats in the future.
+
+In other words, AutoQASM is a quantum programming interface built in Python.
+OpenQASM is a quantum assembly language, often used as a serialization format
+for quantum programming frameworks and quantum hardware providers. We can
+represent a quantum program equivalently in either format, but using AutoQASM
+allows one to also make use of Python, including the Amazon Braket SDK.
+
+### 3. What is the relationship between AutoQASM and the Amazon Braket SDK?
+
+AutoQASM lives alongside the Amazon Braket SDK as an experimental feature
+branch. It supplements the program building experience and integrates with
+Amazon Braket SDK features. For instance, one can build a program through
+AutoQASM, using Amazon Braket SDK device objects, and then run the program
+with the SDK.
```

Comment:
```suggestion
AutoQASM, and then use the SDK to run the program on a local simulator or on
an Amazon Braket device.
```

Reply:
I was trying to include an example of the benefit of integrating with the BDK beyond running on a device, but maybe we don't have enough functionality there yet